<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAppListClass</class>
 <widget class="QWidget" name="QAppListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAppList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filterexp">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RefreshBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="procssTableView"/>
   </item>
   <item>
    <widget class="QTableView" name="appTableView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qapplist.qrc"/>
 </resources>
 <connections/>
</ui>
